docs/source/release/v6.6.0/Direct_Geometry/General/Bugfixes/34564.rst

```diff
@@ -0,0 +1 @@
+A crash is now avoided when extracting a tube while an invalid axis is selected.
```

Comment:
Does this release note need to mention which interface this relates to ie ALFView?